CRITICAL - 2022-03-22 09:00:14 --> Argument 1 passed to view() must be of the type string, null given, called in C:\xampp\htdocs\project\codeigniter4\app\Views\layouts\auth.php on line 23
#0 C:\xampp\htdocs\project\codeigniter4\app\Views\layouts\auth.php(23): view(NULL)
#1 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\project\codeigniter4\system\Common.php(1089): CodeIgniter\View\View->render('layouts/auth', Array, true)
#4 C:\xampp\htdocs\project\codeigniter4\app\Controllers\Login.php(13): view('layouts/auth')
#5 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(832): App\Controllers\Login->index()
#6 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\project\codeigniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\project\codeigniter4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-22 09:00:53 --> Argument 1 passed to view() must be of the type string, null given, called in C:\xampp\htdocs\project\codeigniter4\app\Views\layouts\auth.php on line 23
#0 C:\xampp\htdocs\project\codeigniter4\app\Views\layouts\auth.php(23): view(NULL)
#1 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\project\codeigniter4\system\Common.php(1089): CodeIgniter\View\View->render('layouts/auth', Array, true)
#4 C:\xampp\htdocs\project\codeigniter4\app\Controllers\Login.php(13): view('layouts/auth')
#5 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(832): App\Controllers\Login->index()
#6 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\project\codeigniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\project\codeigniter4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-22 09:02:37 --> Call to undefined method CodeIgniter\View\View::view()
#0 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(210): include()
#1 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\project\codeigniter4\system\Common.php(1089): CodeIgniter\View\View->render('layouts/auth', Array, true)
#3 C:\xampp\htdocs\project\codeigniter4\app\Controllers\Login.php(13): view('layouts/auth')
#4 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(832): App\Controllers\Login->index()
#5 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\project\codeigniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\project\codeigniter4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-03-22 09:02:37 --> Call to undefined method CodeIgniter\View\View::view()
#0 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(210): include()
#1 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\project\codeigniter4\system\Common.php(1089): CodeIgniter\View\View->render('layouts/auth', Array, true)
#3 C:\xampp\htdocs\project\codeigniter4\app\Controllers\Login.php(13): view('layouts/auth')
#4 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(832): App\Controllers\Login->index()
#5 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\project\codeigniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\project\codeigniter4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-03-22 09:02:42 --> Argument 1 passed to view() must be of the type string, null given, called in C:\xampp\htdocs\project\codeigniter4\app\Views\layouts\auth.php on line 23
#0 C:\xampp\htdocs\project\codeigniter4\app\Views\layouts\auth.php(23): view(NULL)
#1 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\project\codeigniter4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\project\codeigniter4\system\Common.php(1089): CodeIgniter\View\View->render('layouts/auth', Array, true)
#4 C:\xampp\htdocs\project\codeigniter4\app\Controllers\Login.php(13): view('layouts/auth')
#5 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(832): App\Controllers\Login->index()
#6 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\project\codeigniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\project\codeigniter4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-03-22 09:32:16 --> Error connecting to the database: Unknown database 'ci4starterkit'
CRITICAL - 2022-03-22 09:32:16 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'ci4starterkit'
#0 C:\xampp\htdocs\project\codeigniter4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\project\codeigniter4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\project\codeigniter4\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\project\codeigniter4\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\project\codeigniter4\app\Controllers\Login.php(23): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(832): App\Controllers\Login->auth()
#6 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\project\codeigniter4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\project\codeigniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\project\codeigniter4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
